docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,337 @@
+   "B" for "Bionic"). If a patch is to be applied to multiple releases a
+   list of release names can be provided.
+
+   Examples::
+
+     [SRU][Focal][PULL] Focal update: v5.4.56 upstream stable release
+     [SRU][B][F][PATCH 1/1] KVM: fix overflow of zero page refcount with ksm running
+
+   .. note::
+
+      * If the patch has to be applied to multiple releases and the subject
+        becomes to hard to follows, you can feel free to drop some of the
+        brackets. For example::
+
+         [SRU][B][F][J][PATCH 0/1] UBUNTU: [Config]: Add support for modifying LDT
+
+        can easily become::
+
+         [SRU][B,F,J][PATCH 0/1] UBUNTU: [Config]: Add support for modifying LDT
+
+        or::
+
+         [SRU][B/F/J][PATCH 0/1] UBUNTU: [Config]: Add support for modifying LDT
+
+      * If the patch has to be applied to a specific derivative for multiple
+        release this is the recommended format::
+
+         [SRU][B:linux-kvm,F:linux-kvm] [PATCH 0/1] UBUNTU: [Config] kvm: Add support for modifying LDT
+
+#. If the patch requested doesn't come from upstream it must contain one of
+   the following on the Subject line after the release name string
+   described previously:
+
+   .. list-table:: Example table
+      :header-rows: 1
+
+      * - Descriptor
+        - Meaning
+      * - UBUNTU: [Packaging]
+        - This is a patch to the Ubuntu Packaging, including the contents
+          of the various debian* directories (where not covered by a more
+          specific categorisation)
+      * - UBUNTU: [Config]
+        - This is an update to the kernel configuration as recorded in the
+          debian.<branch>/config directory. Please check
+          ``debian.master/config/README.rst`` or `here
+          <https://discourse.ubuntu.com/t/kernel-configuration-in-ubuntu/35857>`__
+          for more information about the config format.
+      * - UBUNTU: ubuntu
+        - This is an update to an Ubuntu specific driver in the ubuntu/
+          directory.
+      * - UBUNTU: SAUCE:
+        - This is a patch to the kernel code that has not been applied on
+          mainline (Linus' tree). This category covers the following cases:
+
+             #. The submitter has either authored the patch or obtained the
+                patch from a non-upstream source.
+             #. The patch has been applied to an upstream tree but not yet
+                merged on mainline.
+             #. The patch is never expected to be submitted upstream but is of
+                enough value for Ubuntu to carry it in our tree.
+             #. The patch has been submitted to upstream but is of enough
+                value for Ubuntu to carry it in our tree regardless of
+                upstream acceptance.
+
+      * - UBUNTU: [Debian]
+        - This patch changes the debian rules files (``debian/rules,
+          debian*/rules.d/*`` and possibly ``debian*/control`` and
+          ``debian*/control.d/*``).
+
+   Example::
```

Comment:
I think a description of what this example is trying to demonstrate would help.

Reply:
After talking to Stefan, it turned out that this can be dropped. Thanks!